### ISSUE TITLE
Not able to navigate to classes after recent updates (base 3.0.1)

### ISSUE THREAD

**Issue by curtisblackwell** [php-integrator/atom-navigation#41]
I updated all of the php-integrator packages yesterday. Now when I <kbd>cmd</kbd> + click on class names, nothing happens.

I've tried force indexing a project, but that doesn't seem to do anything. I removed it from `projects.cson` and added it back and indexed it with the UI, no change.

Package  | Version
:- | -:
php-integrator-base | 3.0.1
php-integrator-navigation | 1.2.1
hyperclick | 0.1.2

**Comment by Gert-dev** [comment]
Does this happen for your own classes as well or only for built-in classes? In the case of built-in classes this may be related to https://github.com/php-integrator/atom-base/issues/305 .

There was also the problem of the latest version of this package not being properly published for Atom, causing navigation to effectively disable itself, though I see you already have 1.2.1, so that shouldn't be an issue anymore.

**Comment by curtisblackwell** [comment]
TL;DR I uninstalled/reinstalled everything and it's working.

I wasn't getting those errors, but no functionality from php-integrator seemed to be working (had all 5 packages installed and up-to-date). I was going through the steps in php-integrator/atom-base#305, and wasn't finding any issues. I tried deleting the indexes for a single project and re-indexing, but the re-index command wouldn't do anything. Uninstalled all php-integrator packages, reinstalled them all, now everything works.

**Comment by Gert-dev** [comment]
Good to hear!

In the past it sometimes happened that Atom didn't install the entire package properly or that some files seemed to have disappeared from the PHP install, though the latter was before Composer was used, so I suppose that shouldn't be happening anymore.
